pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -211,19 +212,19 @@ contract MerkleOrchard is IDistributor {
      * These will be pulled from the user
      */
     function seedAllocations(
```

Comment:
The docs for this function still say 'rewarder'.

Reply:
changed in forthcoming PR